### PR TITLE
Update pull request guide and separate documentation guide for the project_euler

### PR DESCRIPTION
**Pull request by iattempt**
**Fixes issue:**

**Changes:**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -6,6 +6,18 @@
...
+     https://github.com/OpenGenus/cosmos/tree/master/guides/coding_style
+
      Note: A coding style guide may not exist for your language, since this is still in beta.
 -->
+
+<!-- Make sure to look at the Documentation Style Guide in guides/documentation.md:
+
+     https://github.com/OpenGenus/cosmos/blob/master/guides/documentation_guide.md
+
+     The document style guide may not apply for your algorithm category, you must also look at specified guide under all of the directory in the category, e.g., for the euler project:
```
Comment (arnavb) [inline .github/PULL_REQUEST_TEMPLATE.md]: e.g, for project euler: :laughing:

File: code/online_challenges/src/project_euler/documentation_guide.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Template
```
Comment (arnavb) [inline code/online_challenges/src/project_euler/documentation_guide.md]: The Title should be Documentation Guide

**Comment by iattempt** [comment]
@arnavb thanks for review, I have fixed the suggests.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -6,6 +6,18 @@
...
+     https://github.com/OpenGenus/cosmos/tree/master/guides/coding_style
+
      Note: A coding style guide may not exist for your language, since this is still in beta.
 -->
+
+<!-- Make sure to look at the Documentation Style Guide in guides/documentation.md:
+
+     https://github.com/OpenGenus/cosmos/blob/master/guides/documentation_guide.md
+
+     The document style guide may not apply for your algorithm category, you must also look at specified guide under all of the directory in the category, e.g., for the project euler:
```
Comment (arnavb) [inline .github/PULL_REQUEST_TEMPLATE.md]: Remove _the_ in project euler

**Comment by iattempt** [comment]
@AdiChat Please merge.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**